packages/mdsvex/src/globals.d.ts

```diff
@@ -0,0 +1,17 @@
+declare module '*.svx' {
+	import type { SvelteComponentDev } from 'svelte/internal';
+
+	export default class Comp extends SvelteComponentDev {
+		$$prop_def: {};
+	}
+	export const metadata: Record<string, any>;
+}
+
+declare module '*.svelte.md' {
+	import type { SvelteComponentDev } from 'svelte/internal';
+
+	export default class Comp extends SvelteComponentDev {
+		$$prop_def: {};
+	}
+	export const metadata: Record<string, any>;
+}
```

Comment:
Is this extension a defacto standard that people use?

Reply:
It was listed in svelte-add mdsvex starter, that's why I added it. Not sure practically if people use it, though. I think would be nice to leave it in. Your call.